### ISSUE TITLE
Modify the `aria-label`s for pending organizations to contain the pending org's name

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5130]
## Area or Component
Support admin > (Click "Organizations pending identify verification") > Edit or verify organization identity

## Quick Summary
The "Edit/Verify" button as well as the trash can icon (delete button) for the pending organization need to have a more descriptive `aria-label` so that a user using a screenreader can tell which org it is for.

### How to Replicate
Use voice over to navigate a pending organization's "Edit/Verify" button and trash icon button. You will hear no context for which org this is for.

## Accessibility Violation
[Name: The element's name is missing or incorrect](https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect)

## Action Requested
Add the pending organization name to the `aria-label` for "Edit/Verify" button and the trash icon.

## Source Code (optional)

[which file or line is the issue attached to?]: #
